### DISCUSSION TITLE
can't I just dd an image to the USB stick and I am all set up?

### DISCUSSION THREAD

**Discussion by harridu** [ventoy/Ventoy#2663] [Ideas]
It would be nice to have an image to dd to an USB stick. Something that requires root just when needed.

**Comment by steve6375** [comment]
dd would require different sized images for different sized drives.
e.g. If I had a 2TB image, which had 800GB of ISO files on it, I would still need to write 2TB of data which would take many hours.
Can you explain exactly what you mean?
